Gemfile.lock

```diff
@@ -258,7 +258,7 @@ GEM
       eventmachine (>= 0.12.9)
       http_parser.rb (~> 0.5.3)
     erubis (2.7.0)
-    eventmachine (1.0.8)
+    eventmachine (1.2.3)
```

Comment:
Yep, sure. It just feels weird because it is not a direct dependency of the project. It's only required by guard-rails. Anyway, I'll specify the version in the Gemfile for now...